### PR TITLE
[DDO-1524] Uptime check labels

### PR DESCRIPTION
**Pull request by jack-r-warren**
Allow this module to add labels to its alert policy so that Revere can understand what the alert policy actually means for a given service / instance. Verified working via https://github.com/broadinstitute/terraform-ap-deployments/pull/410

### CODE REVIEW

**Review by knapii-developments — APPROVED** [review]

File: terraform-modules/stackdriver/uptime-check/variables.tf [hunk 1]
```diff
@@ -30,11 +30,76 @@ variable "notification_channels" {
...
+  default     = null
+  description = <<-EOT
+    Override the environment of the service this alert monitors as understood by
+    [Revere](https://github.com/broadinstitute/revere).
+
+    By default, uses the last part of `var.service` (splitting on hyphens).
+  EOT
+}
+
+variable "revere_service_degredation" {
```
Comment (choover-broad) [inline terraform-modules/stackdriver/uptime-check/variables.tf:79]: nit: typo (degradation)

File: terraform-modules/stackdriver/uptime-check/variables.tf [hunk 2]
```diff
@@ -30,11 +30,76 @@ variable "notification_channels" {
...
+  type        = string
+  default     = "uptime"
```
Comment (choover-broad) [inline terraform-modules/stackdriver/uptime-check/variables.tf:81]: hm, again a bit confused about other potential values here

**Review by choover-broad — APPROVED** [review]
lgtm, nice! Like the use of labels here

File: terraform-modules/stackdriver/uptime-check/variables.tf (hunk 1 above)
Comment (jack-r-warren) [reply terraform-modules/stackdriver/uptime-check/variables.tf:79]: only the most embarrassing typo everywhere, I can't imagine. Thank you lol

File: terraform-modules/stackdriver/uptime-check/variables.tf (hunk 2 above)
Comment (jack-r-warren) [reply terraform-modules/stackdriver/uptime-check/variables.tf:81]: I think we'll probably change this based on discussion in https://github.com/broadinstitute/revere/pull/15

**Review by choover-broad — APPROVED** [review]
Thanks for clarifying how the alert type variable works!

**Pull request merged**